tests/support/globalSetup.ts

```diff
@@ -64,6 +66,14 @@ export default async function globalSetup(_config: FullConfig): Promise<void> {
   } else {
     await startPalmHttpd();
   }
+
+  return async () => {
+    if (teardownHappened) {
+      throw new Error("GlobalTeardown called twice");
+    }
+    teardownHappened = true;
+    killAllProcesses();
```

Comment:
`killAllProcesses` is idempotent. (In fact it is actually called twice: once here and in the exit hook). So no need to guard this.

```suggestion
    killAllProcesses();
```